### ISSUE TITLE
Using `method="GET"` breaks page

### ISSUE THREAD

**Issue by SIMULATAN** [huntabyte/formsnap#95]
## Impact
When setting `method="GET"` on the `Form.Root`, half the site broke for me. Routing stopped working (not even the `$page` store was updated), buttons relying on said store didn't appear, etc.

## Reason
This is due to the library running `use:enhance` even when the method isn't `POST` which is unsupported by svelte.
![picture of the error in the console](https://github.com/huntabyte/formsnap/assets/63104422/9a5c514a-7651-42e6-a9b3-26de91fcecf9)

## Workaround
Set `asChild` on the `Root` element and delegate rendering to your own `<form>`.
Example:
```html
<Form.Root asChild let:attrs method="GET" ...>
	<form {...attrs}>
		...
	</form>
</Form.Root>
```
*I hope this workaround saves you from the tens of minutes of debugging I went through*

**Comment by fstodulski** [comment]
@SIMULATAN Perfect! thank you for a workaround!

**Comment by minmur** [comment]
Yea, workaround works. But looks so so... :/

**Comment by huntabyte** [comment]
We're going to be rewriting formsnap to support Superforms v2 and this issue will be addressed then! Apologies!

**Comment by SIMULATAN** [comment]
> We're going to be rewriting formsnap to support Superforms v2 and this issue will be addressed then! Apologies!

Great to hear! Thank you for your work on this awesome library :)

**Comment by huntabyte** [comment]
Closed by #131
